index.d.ts

```diff
@@ -4,6 +4,7 @@ import https = require('https');
 import http = require('http');
 import events = require('events');
 import Stream = require('stream');
+import { HandleFunction } from 'connect';
```

Comment:
This is just a type import, connect itself is not a dependency. But it's better than just a Function type.

Reply:
This now causes a typescript compilation error downstream since the `@types/connect` is in DevDependencies instead of Dependencies.  I added https://github.com/tedeh/jayson/issues/150 for this issue.